index.html

```diff
@@ -121,6 +131,28 @@ <h5>About Myself</h5>
             </ul>
         </div>
     </section>
+
+    <footer class="footer">
+        <div class="footerhouse">
+            <section class="footer-section">
+                <div class="footercontainer">
+                    <h6>Contact me</h6>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a
+                        project
+                        that needs coding. I'd love to help with it!</p>
+                </div>
+            </section>
+            <form action="https://formspree.io/f/xlekzbpj" action="post" class="form">
```

Comment:
- You have done well using the action method for your form, but according to the project requirements, your form should have a method attribute equal to POST. Please fix this to enable proper submission of your form 👍

Reply:
Noted with thanks @mckent05